### ISSUE TITLE
Alternative possible names for X

### ISSUE THREAD

**Issue by danrobinson** [dodger487/dplython#28]
Have you considered `_`? IMHO it feels more evocative (of member variables, for example) and invisible. The biggest downside is that it already has a meaning in the interactive interpreter (the last evaluated expression). If people want to use dplython in an interactive session and use the _ for that purpose, they could use `from dplython import _ as X`.

Are there any other names you considered? Assuming you want it to be a single character, I guess there are only 53 possibilities. I could see a case for x (lowercase is subtler and easier to type), d (for dataframe), s (for self), or c (for column).

**Comment by dodger487** [comment]
When I first wrote this, I actually used `_`. Like you mention, the big downside is use in the ipython notebook. I think this is a big enough issue to avoid using it. Since dplython is probably best for exploratory data analysis, I imagine a lot of people will use it interactively in the console. There was also an issue with using wildcard imports where `_` doesn't get imported. Sure, wildcard imports are discouraged, but people still use them.

Perhaps the best way to think about a good choice of Manager is a a bunch of desired qualities.
1.  Easy to type: avoid using shift or alternating with shift and non-shift characters, or strange parts of keyboard
2. Quick to type: one or two characters
3. Doesn't clash with variables users might typically use
4. Easily recognizable
5. "Makes sense": A name like `DATA` makes more sense than `foo`. 

I'm waiting for inspiration to strike but the top candidates are probably still `X`, `x`, and `d`. Maybe `df`, though that probably violates 3. I worry about 3. for `x` as well, but it does save holding down the shift key, which is pretty annoying given that the next key will almost invariably be `.`

Also, right now a user can use whatever they want for `X`. It's just a matter of calling

```
from dplython import Manager
x = Manager()
```
